esy.json

```diff
@@ -1,8 +1,20 @@
 {
+  "esy": {
+    "build": "make build",
+    "buildsInSource": "_build"
```

Comment:
Why do we need this line?

Reply:
That line is so the esy build location is consistent with normal dune. The `main` is specified to be `_build/default/src/extension.bc.js` in the package.json. If that line wasn't there, it would be built under the `_esy` folder.